bucket/camstudio.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2.7.4-r354",
+    "description": "Free desktop recorder and video streaming software",
+    "homepage": "https://camstudio.org/",
+    "license": "GPL-3.0-or-later",
+    "url": [
+        "https://eu2.contabostorage.com/a4054a9f5df24ecf87c4a33ac9b6477b:camstudio/CamStudioSetup.exe#/dl.exe",
```

Comment:
Any difference between this link and the other one on the homepage: https://cs07112018.s3.us-east-2.amazonaws.com/CamStudioSetup.exe ?

Reply:
Not sure, but I guess they are just different download locations.
One of the server is in the US, and another in Europe.